packages/client/src/service/skeleton.ts

```diff
@@ -850,14 +889,100 @@ export class Skeleton extends MetaDBManager {
+      this.chain.blocks.latest?.hash() ?? 'na'
+    )}`
+
+    forceShowInfo = forceShowInfo ?? false
+    lastStatus = lastStatus ?? status
+
+    if (forceShowInfo || status !== lastStatus) {
+      let beaconSyncETA = 'na'
+      if (!this.status.linked && subchain0 !== undefined) {
+        // Print a progress report making the UX a bit nicer
+        let left = this.bounds().tail - BIGINT_1 - this.chain.blocks.height
+        if (this.status.linked) left = BIGINT_0
+        if (left > BIGINT_0) {
+          if (this.pulled === BIGINT_0) {
+            this.config.logger.info(`Beacon sync starting left=${left}`)
+          } else {
+            const sinceStarted = (new Date().getTime() - this.started) / 1000
+            beaconSyncETA = `${timeDuration((sinceStarted / Number(this.pulled)) * Number(left))}`
+            this.config.logger.debug(
+              `Syncing beacon headers downloaded=${this.pulled} left=${left} eta=${beaconSyncETA}`
+            )
+          }
+        }
+      }
+
+      let logInfo
+      if (isValid) {
+        logInfo = `vm = cl = ${chainHead}`
+      } else {
+        logInfo = `vm=${vmHead?.header.number} hash=${short(
+          vmHead?.hash() ?? 'na'
+        )} executing=${executing}`
+
+        const beaconsync = this.filling ? 'filling' : fetching === true ? 'fetching' : 'na'
+        // if not synced add subchain info
+        if (!isSynced) {
+          logInfo = `${logInfo} cl=${subchain0?.head} ${chainHead}`
+          const subchainLen = this.status.progress.subchains.length
+          logInfo = `${logInfo} subchains(${subchainLen}) linked=${
+            this.status.linked
+          } ${this.status.progress.subchains
+            // if info log show only first subchain to be succinct
+            .slice(0, forceShowInfo ? 1 : this.status.progress.subchains.length)
+            .map((s) => `[head=${s.head} tail=${s.tail} next=${short(s.next)}]`)
+            .join(',')}${subchainLen > 1 ? '…' : ''} beaconsync=${beaconsync} ${
+            beaconSyncETA !== undefined ? 'eta=' + beaconSyncETA : ''
+          } reorg chain=${
+            this.status.canonicalHeadReset &&
+            (subchain0?.tail ?? BIGINT_0) <= this.chain.blocks.height
+          }`
+        } else {
+          logInfo = `${logInfo} cl = ${chainHead}`
+        }
+      }
+      peers = peers !== undefined ? `${peers}` : 'na'
+      this.config.logger.info(`${logPrefix}: ${status} ${logInfo} peers=${peers}`)
+      this.config.logger.info('---------------------------------------')
+    } else {
+      this.config.logger.debug(
+        `${logPrefix} ${status} linked=${
+          this.status.linked
+        } subchains=${this.status.progress.subchains
+          .map((s) => `[head=${s.head} tail=${s.tail} next=${short(s.next)}]`)
+          .join(',')} reset=${this.status.canonicalHeadReset} ${chainHead}`
+      )
+    }
+    return status
```

Comment:
Great sync improvements! 🤩

Some formatting aspects: this line is (far) too long though now, doesn't fit for normal laptop setups any more.

Can we split this in two lines or alternatively do we need *all* the info there? Maybe also a way to present some information only periodically? Just options.

So line should really not go much further than what is now `linked=true`.